frontend/isoko/src/pages/Authentication/SignUp.tsx

```diff
@@ -164,28 +143,51 @@ const Description = styled.div`
    box-shadow: 5px 5px 4px rgba(0, 0, 0, 0.25);
 `;
 
+const FormError = styled.p`
+   justify-content: center;
+   color: red;
+   text-align: left;
+   width: 70%;
+   margin: 0 auto;
+`;
+
 const SignUp: React.FC = () => {
-   const [listBusinessState, setListBusinessState] = useState('');
    const [firstName, setFirstName] = useState('');
    const [lastName, setLastName] = useState('');
    const [email, setEmail] = useState('');
    const [password, setPassword] = useState('');
    const [confirmPassword, setConfirmPassword] = useState('');
+   const [err, setErr] = useState('');
 
    const navigate = useNavigate();
 
    const checkAllFieldsEntered = () => {
       if (
-         listBusinessState.length === 0 ||
          firstName.length === 0 ||
          lastName.length === 0 ||
          email.length === 0 ||
          password.length === 0 ||
          confirmPassword.length === 0
       ) {
+         setErr('Make sure all fields are filled out');
          return false;
       }
-      return true;
+      checkPasswordsMatch();
```

Comment:
if checkAllFieldsEntered is still going to call checkPasswordsMatch (which I don't think it should), the name should be changed to something like checkFormValidity since it checks both password matching and that all the fields are entered.

Reply:
I think the names of the functions could maybe be improved. But I think the chaining of the validation functions works well here. What do you think should be changed about the way its implemented?

Reply:
Implementation is fine RE the other comment, but yeah I think name could be changed for clarity. checkAllFieldsEnteredProperly, checkFormValidty, smth like that but again it is a nit lol